sycl/test/graph/graph-explicit-dotp.cpp

```diff
@@ -68,12 +64,11 @@ int main() {
 
   auto node_c = g.add(
       [&](sycl::handler &h) {
-        h.parallel_for(sycl::range<1>{n},
```

Comment:
We should guard the reduction code behind `#ifdef TEST_GRAPH_REDUCTIONS` like we've done in other tests to be consistent.

Reply:
Done!